tests/EdFi.Common.UnitTests/Security/Pbkdf2HmacSha1SecureHasherTests.cs

```diff
@@ -68,21 +67,27 @@ public class When_computing_an_invalid_hash : TestFixtureBase
 
             protected override void Arrange()
             {
-                _secureHasher = new Pbkdf2HmacSha1SecureHasher(null);
+                _secureHasher = new Pbkdf2HmacSha1SecureHasher();
 
-                _request = new SecureHashRequest { HashAlgorithm = 1234 };
+                _request = new SecureHashRequest {HashAlgorithm = 1234};
```

Comment:
Shouldn't this change the same as the first test, swapping from a SecureHashRequest to just the AlgorithmHashCode and Salt?

Reply:
legacy artifact, i deprecated securehashrequest

Reply:
this line has been removed